warehouse/accounts/views.py

```diff
@@ -91,6 +91,8 @@
 from warehouse.utils.http import is_safe_url
 
 USER_ID_INSECURE_COOKIE = "user_id__insecure"
+REMEMBER_DEVICE_COOKIE = "remember_device"
+REMEMBER_DEVICE_SECONDS = 2592000  # 30 days
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏shouldn't this be retrieved from config?

Reply:
Yeah, I can get this from config.